### DISCUSSION TITLE
Comercial use of images produced by ai2thor

### DISCUSSION THREAD

**Discussion by matthieuo** [allenai/ai2thor#1096] [General]
Hey,

The license of ai2thor is apache, so I guess the 3d assets are also royalty free.
On the other hand, ai2thor uses the unity 3d engine. If we work in a company, do we have to pay a unity license to use the images generated with ai2thor, for example to train machine learning algorithms?

Thanks,

Matthieu

**Comment by dchichkov** [comment]
It'd be great to have clarification on the license of 3D assets used.  Are these distributed under Apache 2.0?

**Comment by StoneT2000** [comment]
Just bumping this thread up. Was wondering if there was a CC license for the objects in the AI2Thor? I noticed a few other orgs/groups have released versions of this dataset like HSSD, but can't find any license information: https://3dlg-hcvc.github.io/hssd/
